lib/gyro/template.rb

```diff
@@ -12,24 +12,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+require 'yaml'
+
 module Gyro
   # Gyro Template Helper
   #
   module Template
+
+    # Print template list representation
+    #
     def self.print_list
-      Gyro::Template.directory.children.sort_by(&:basename).each do |entry|
+      template_config_path = Gyro::Template.directory + 'config.yml'
+      template_config = YAML.load_file(template_config_path)
+      alias_template_hash = template_config['alias']
+      deprecated_template_list = template_config['deprecated']
+
+      alias_template_list = alias_template_hash.map{ |key, value| key }.to_a
```

Comment:
I think that `Hash` has a method `keys` directly?

Reply:
Yes there is